File: CS/Models/EmployeeViewModel.cs
using System.Collections.Generic;
using DevExpress.AspNetCore.Bootstrap;

namespace TagBoxWithCustomTags.Models
{
    public class EmployeeViewModel
    {
        public int Id { get; set; }

        // Selected TagBox' values (both predefined and custom). 
        public BootstrapTagBoxValues<int> Roles { get; set; }
        public IEnumerable<Role> AllRoles { get; set; }
    }
}

File: VB/Models/EmployeeViewModel.vb
Imports System.Collections.Generic
Imports DevExpress.AspNetCore.Bootstrap

Namespace TagBoxWithCustomTags.Models
    Public Class EmployeeViewModel
        Public Property Id() As Integer

        ' Selected TagBox' values (both predefined and custom). 
        Public Property Roles() As BootstrapTagBoxValues(Of Integer)
        Public Property AllRoles() As IEnumerable(Of Role)
    End Class
End Namespace
